utils/package.py

```diff
@@ -9,4 +10,13 @@ def get_package_version() -> Optional[str]:
     except Exception:
         pass
 
-    return None
\ No newline at end of file
+    return None
+
+
+def get_latest_package_version() -> Optional[str]:
+    resp = requests.get('https://pypi.org/pypi/elementary-data/json')
+    try:
+        resp.raise_for_status()
+        return resp.json()['info']['version']
+    except requests.HTTPError:
```

Comment:
Lets make the exception general :)
It could be that in the future pypi will change the response structure and we will get a different error that will break the cli